pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -150,19 +150,19 @@ def start_predicting(self, trainer: 'pl.Trainer') -> None:
         self._results = trainer.run_stage()
 
     def training_step(self, *args, **kwargs):
-        return self.lightning_module.training_step(*args, **kwargs)
+        return self.model.training_step(*args, **kwargs)
 
     def post_training_step(self):
         pass
 
     def validation_step(self, *args, **kwargs):
-        return self.lightning_module.validation_step(*args, **kwargs)
+        return self.model.validation_step(*args, **kwargs)
 
     def test_step(self, *args, **kwargs):
-        return self.lightning_module.test_step(*args, **kwargs)
+        return self.model.test_step(*args, **kwargs)
 
     def predict_step(self, *args, **kwargs):
-        return self.lightning_module.predict_step(*args, **kwargs)
+        return self.model.predict_step(*args, **kwargs)
```

Comment:
Can you give it a quick look again double check that all plugins that don't override these methods actually refer to the right model/wrapper here and it aligns with your intentions?
So in DDP fore example, self.model would be the DistributedDataParallel.

Reply:
Yeah, can't remember the reason for this and I think it's an error, reverted!

Reply:
Ok, I've done some digging. All training type plugins either don't wrap the model (so `self.model == self.lightning_module`) or override the methods to call `self.model`. So I've removed the overrides now and put it back to using `self.model` in the base class :smiley: